micall/tests/test_consensus_aligner.py

```diff
@@ -804,8 +804,27 @@ def test_write_multiple_alignments(projects):
     assert alignments_file.getvalue() == expected_text
 
 
-# noinspection DuplicatedCode
 def test_count_coord_concordance():
+    projects = ProjectConfig()
+    projects.load(StringIO("""\
+    {"genotype_references": {"test-region": {"is_nucleotide": true,"reference": ["AGATTTCGATGATTCAGAAGATAAGCA"]}}}
+    """))
+    aligner = ConsensusAligner(projects)
+    aligner.consensus = "AGATTTCGATGATTCAGAAGATAAGCA"
+    aligner.coordinate_name = 'test-region'
+    aligner.alignments = [AlignmentWrapper(r_st=0, r_en=27, q_st=0, q_en=27, cigar=[[27, CigarActions.MATCH]])]
+
+    expected_concordance_list = [1.0]*len(aligner.consensus)
+
+    concordance_list = aligner.coord_concordance()
+
+    assert len(concordance_list) == len(expected_concordance_list)
+    for i, concordance in enumerate(concordance_list):
+        assert abs((concordance - expected_concordance_list[i])) < 0.01
```

Comment:
You can compare the two lists directly with [`pytest.approx`]. The same suggestion applies in several comparisons below.

[`pytest.approx`]: https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-approx

Reply:
Thanks, much better!